### ISSUE TITLE
Layout issue

### ISSUE THREAD

**Issue by andre2007** [jenkinsci/code-coverage-api-plugin#97]
If the report file names are lengthy, there is a layout issue with the legend (covered / missed)

![image](https://user-images.githubusercontent.com/1451047/61196520-8ddca700-a6cf-11e9-9c10-c9a645199a90.png)

Also another issue: I have for a project ~70 lst files. I have assumed they would be aggregated automatically to 1 report but every lst file is shown as an individual report (see screenshot above).
Is this the intended behavior?

**Comment by cizezsy** [comment]
@andre2007 Oh..... Sorry, it's my fault, I misunderstand your meaning before. 

I have added support for merging multiple reports into one report now: https://github.com/jenkinsci/code-coverage-api-plugin/pull/98. 

You can try with this PR and add a parameter `mergeToOneReport: true` like this:
```
publishCoverage adapters: [coberturaAdapter(mergeToOneReport: true, path: '**/*.xml')]
```

**Comment by andre2007** [comment]
@cizezsy 
Thanks. Is there a possibility to set the default value for ```mergeToOneReport``` programmatically within a coverage adapter?

(In 99% the default value should be ```true``` for LST reports)

**Comment by cizezsy** [comment]
@andre2007 Yes, you can set it in the constructor method:

For example:
```
public LSTReport(String path) {
     setMergeToOneReport(true);
}
```
But this default value will be overridden by Jenkinsfile. If you want to set it to always true, you might need to override the 'isMergeToOneReport()' method and return true(But I am not sure if it is a good idea).

**Comment by andre2007** [comment]
@cizezsy thanks, works great!

**Comment by cizezsy** [comment]
Fixed in version 1.0.12.

**Comment by andre2007** [comment]
@cizezsy by setting ```setMergeToOneReport``` to ```true``` in the coverage reporter constructor I would assume following behavior: While creating new post build action with coverage type "LSTReport", the checkbox "Merge to one report" should be ticked by default.
But this is not the case. Is this a bug? Or what do you mean with "overriden by Jenkinsfile"?
